iOS Client/PodcastsController/PodcastFilterTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PodcastFilterTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class PodcastFilterTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var toggleSwitch: UISwitch!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var podcastImage: UIImageView!
+    
+    var podcast: PodcastStream?
+    
+    static var reuseIdentifier: String = "PodcastFilterTableViewCell"
+    func configure(image: UIImage? = nil, podcast: PodcastStream, on: Bool = false) {
+        toggleSwitch.isOn = on
+        titleLabel.text = podcast.title
+        podcastImage.image = image
+        self.podcast = podcast
+        podcastImage.layer.cornerRadius = 10
```

Comment:
can this be done just once in `awakeFromNib`?

Reply:
does `podcastImage.layer` get reset when `podcastImage.image` changes? I'm not 100% sure how that works